### PR TITLE
Gep 24

### PR DESCRIPTION
**Pull request by eglynas**
Registration form created with call to back + checked response
B: Toast behind a popup if registration data is incorrect

### CODE REVIEW

**Comment by TomasKireilis** [comment]
image

**Comment by TomasKireilis** [comment]
kai errorai nerodo erroro popupo

File: src/AcountPopups/RegisterAccountPopup.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+            <Row>
+              <Form className="account-registration-form">
+                <Form.Group>
+                  <Form.Label id="email">Elektroninio pašto adresas</Form.Label>
+                  <Form.Control
+                    type="text"
+                    placeholder="pavyzdys@pastas.lt"
+                    onChange={(event) => {
+                      registrationData.email = event.target.value;
+                      setRegistrationData(registrationData);
```
Comment (TomasKireilis) [inline src/AcountPopups/RegisterAccountPopup.js:138]: truksta check()

File: src/App.css [hunk 2]
```diff
@@ -430,7 +446,7 @@ table thead tr {
 .toast-notification {
   left: 50%;
   transform: translate(-50%);
-  z-index: 5000;
+  z-index: 200000 !important;
```
Comment (TomasKireilis) [inline src/App.css]: cia bent kazka daro? jei ne kam keist?

**Review by TomasKireilis — CHANGES_REQUESTED** [review]

**Review by mdovile — CHANGES_REQUESTED** [review]
Ar dar aktyvus šitas PR? Reikia Tomo komentarus sutvarkyti

File: src/AcountPopups/RegisterAccountPopup.js [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+  const registerAccount = async() => {
+    let data = {
+      username: registrationData.email,
+      password: registrationData.password,
+      fullName: registrationData.fullName,
+      address: registrationData.address,
+      phoneNumber: registrationData.phoneNumber,
+    };
+    let response = await postUser(data);
+    console.log(response);
```
Comment (mdovile) [inline src/AcountPopups/RegisterAccountPopup.js]: Console log ištrinti

File: src/Service/RegistrationService.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+export async function postUser(data) {
+    const requestOptions = {
+        method: "POST",
+        headers: { "Content-Type": "application/json" },
+        body: JSON.stringify(data),
+    };
+    const response = await fetch("http://localhost:8080/register", requestOptions);
+    console.log(response.status);
```
Comment (mdovile) [inline src/Service/RegistrationService.js]: Ištrinti

**Review by mdovile — CHANGES_REQUESTED** [review]

**Review by TomasKireilis — APPROVED** [review]

**Review by mdovile — APPROVED** [review]

**Pull request merged**